src/components/SearchableMultiSelect/SearchableMultiSelect.jsx

```diff
@@ -547,14 +561,15 @@ const SearchableMultiSelect = createClass({
 			<div
 				{...omitProps(passThroughs, SearchableMultiSelect)}
 				className={cx('&', className)}
+				Error={errorChildProps}
```

Comment:
Let's leave this out since `div` won't accept this prop and will probably throw a React warning about it in the console.

Reply:
where would you suggest moving the error props? I wrapped everything in a div so I could separately render out the error text underneath the DropMenu component